pkg/scheduler/preemption/preemption_test.go

```diff
@@ -1345,6 +1347,232 @@ func TestPreemption(t *testing.T) {
 	}
 }
 
+func TestFairPreemption(t *testing.T) {
```

Comment:
What is the coverage for the `fairPreemptions` function?

Reply:
It's fully covered now.